monai/transforms/post/array.py

```diff
@@ -46,3 +48,72 @@ def __call__(self, img, to_onehot=None, num_classes=None):
             outputs.append(img[:, i : i + 1])
 
         return outputs
+
+
+class AddActivations(Transform):
+    """
+    Add activation operations to the model output, typically `Sigmoid` or `Softmax`.
+
+    Args:
+        add_sigmoid (bool): whether to add sigmoid function to model output before transform.
+        add_softmax (bool): whether to add softmax function to model output before transform.
+        other (Callable): callable function to execute other activation layers, for example:
+            `other = lambda x: torch.tanh(x)`
+
+    """
+
+    def __init__(self, add_sigmoid=False, add_softmax=False, other=None):
```

Comment:
could replace `add_sigmoid` with `sigmoid`, `add_softmax` with `softmax`, and `AddActivations` with `Activations`, because I feel it's a bit confusing given we have `AddChannel`

Reply:
updated in the latest patch.
Thanks.